roles/DCOS.bootstrap/tasks/main.yml

```diff
@@ -199,9 +199,32 @@
   when: "dcos['version'] is version('1.9', '>=')"
   loop: "{{ upgrade_dir_hash.results }}"
 
-# vvv serve files via http vvv
 
-- name: Serve bootstrap files via dockerized httpd
+#### Docker proxy stuff
+- name: create dropin directory
+  file:
+    path: "/etc/systemd/system/docker.service.d"
```

Comment:
so I do not think this will work on RHEL. Which leads me to the point that we need a test for it.